integration-test/1016-missing-localized-names.py

```diff
@@ -1,14 +1,13 @@
a': u'Nova Caesarea', u'name:uk': u'\u041d\u044c\u044e-\u0414\u0436\u0435\u0440\u0441\u0456', u'name:hy': u'\u0546\u0575\u0578\u0582 \u054b\u0565\u0580\u057d\u056b', u'name:ru': u'\u041d\u044c\u044e-\u0414\u0436\u0435\u0440\u0441\u0438', u'name:ht': u'Nou J\xe8ze', u'name:ur': u'\u0646\u06cc\u0648 \u062c\u0631\u0633\u06cc', u'name:sr': u'\u040a\u0443 \u040f\u0435\u0440\u0437\u0438', u'name:nv': u'Niij\xe9\xedzii Hahoodzo', u'name:ang': u'N\u012bwe C\u0113sar\u0113a', u'ref:fips': u'34', u'name:lad': u'Nueva Jersey', u'name:eo': u'Nov-\u0134erzejo', u'name:en': u'New Jersey', u'name:uz': u"N'yu-Jersi", u'name:gd': u'Jersey Nuadh', u'name:zh': u'\u65b0\u6cfd\u897f\u5dde', u'name:gn': u'Pyahu Jersey', u'name:gv': u'Jersee Noa', u'name:bg': u'\u041d\u044e \u0414\u0436\u044a\u0440\u0441\u0438', u'name:yi': u'\u05e0\u05d9\u05d5 \u05d6\u05e9\u05e2\u05e8\u05e1\u05d9', u'wikidata': u'Q1408', u'name:el': u'\u039d\u03b9\u03bf\u03cd \u03a4\u03b6\u03ad\u03c1\u03c3\u03b5\u03ca', u'name:es': u'Nueva Jersey'}),dsl.way(-61320, wkt_loads('POLYGON ((-74.058837890625 40.65177013430829, -74.0557389479016 40.65175995493708, -74.05332209032987 40.65563874006117, -74.04785156249999 40.65563874006117, -74.04785156249999 40.6473035625225, -74.058837890625 40.6473035625225, -74.058837890625 40.65177013430829))'), {u'name:pt': u'Nova Iorque', u'name:az': u'Nyu-York', u'name:cy': u'Efrog Newydd', u'name:ko': u'\ub274\uc695', u'name:cv': u'\xc7\u0115\u043d\u0115 \u0419\u043e\u0440\u043a', u'name:ik': u'Tchiaq York', u'name:ar': u'\u0646\u064a\u0648\u064a\u0648\u0631\u0643', u'ISO3166-2': u'US-NY', u'wikidata': u'Q1384', u'alt_name': u'New York State', u'way_area': u'2.63588e+11', u'name:ru': u'\u041d\u044c\u044e-\u0419\u043e\u0440\u043a', u'name:pl': u'Nowy Jork', u'name:ta': u'\u0ba8\u0bbf\u0baf\u0bc2 \u0baf\u0bcb\u0bb0\u0bcd\u0b95\u0bcd', u'name:ca': u'Nova York', u'ref:fips': u'36', u'name:ka': u'\u10dc\u10d8\u10e3-\u10d8\u10dd\u10e0\u10d9\u10d8', u'name:haw': u'Nuioka', u'old_name:vi': u'N\u1eefu \u01af\u1edbc', u'wikipedia': u'en:New York', u'source': u'openstreetmap.org', u'boundary': u'administrative', u'name:zh': u'\u7ebd\u7ea6\u5dde', u'name:nah': u'Yancu\u012bc York', u'admin_level': u'4', u'name:lv': u'\u0145ujorka', u'name:ja': u'\u30cb\u30e5\u30fc\u30e8\u30fc\u30af\u5dde', u'name:chr': u'\u13c4\u13f2\u13a9', u'name:he': u'\u05e0\u05d9\u05d5 \u05d9\u05d5\u05e8\u05e7', u'is_in:country_code': u'US', u'name:ga': u'Nua-Eabhrac', u'name:la': u'Novum Eboracum', u'name:uk': u'\u041d\u044c\u044e-\u0419\u043e\u0440\u043a', u'name:bg': u'\u041d\u044e \u0419\u043e\u0440\u043a', u'name:sr': u'\u040a\u0443\u0458\u043e\u0440\u043a', u'name:nv': u'N\xed\xedy\xf3\xf3 Hahoodzo', u'name:el': u'\u039d\u03ad\u03b1 \u03a5\u03cc\u03c1\u03ba\u03b7', u'name:eo': u'Nov-Jorkio', u'name:en': u'New York', u'name': u'New York', u'name:gd': u'Eabhraig Nuadh', u'ref': u'NY', u'name:gv': u'York Noa', u'name:yi': u'\u05e0\u05d9\u05d5 \u05d9\u05d0\u05e8\u05e7', u'name:es': u'Nueva York'}))  # noqa
```

Comment:
What does `# noqa` mean here in the comment (and every other fixture added)?

Reply:
It's how we tell [`flake8` to ignore the line](http://flake8.pycqa.org/en/latest/user/violations.html#in-line-ignoring-errors), otherwise these all end up much longer than the permitted line width and the test fails `flake8` before it has even been run!

I'd love to reformat these so that they are OK (e.g: as I did in the [manually-edited island tests](https://github.com/tilezen/vector-datasource/pull/1583/commits/3cc51e4b049cbd2a08d39744cbbc7dd70885fa18)), but all of the auto-formatting tools I've tried have produced results that look weird to me. And reformatting by hand is a fair bit of work.